tests/gql_behave/tests/memgraph_V1/features/memgraph_exists.feature

```diff
@@ -0,0 +1,67 @@
+Feature: WHERE exists
+
+  Scenario: Test exists with empty edge and node specifiers
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two)
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[]-()) RETURN n.prop;
+          """
+      Then the result should be:
+          | n.prop |
+          | 1      |
+
+  Scenario: Test exists with edge specifier
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two)
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[:TYPE]-()) RETURN n.prop;
+          """
+      Then the result should be:
+          | n.prop |
+          | 1      |
+
+  Scenario: Test exists with wrong edge specifier
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two)
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[:TYPE2]-()) RETURN n.prop;
+          """
+      Then the result should be empty
+
+  Scenario: Test exists with correct edge direction
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two)
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[:TYPE]->()) RETURN n.prop;
+          """
+      Then the result should be:
+          | n.prop |
+          | 1      |
+
+  Scenario: Test exists with wrong edge direction
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two)
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)<-[:TYPE]-()) RETURN n.prop;
+          """
+      Then the result should be empty
```

Comment:
You can add two more tests, test with second node correct label, and incorrect label:

```
MATCH (n:One) WHERE exists((n)-[:TYPE]->(:Five)) RETURN n.prop;
MATCH (n:One) WHERE exists((n)-[:TYPE]->(:Two)) RETURN n.prop;
```

Reply:
Agreed

Reply:
I will not address this here, as it is part of filtering PR, T1236